.editorconfig

```diff
@@ -13,6 +13,7 @@ insert_final_newline = true
 trim_trailing_whitespace = false
 
 [*.fs]
+fsharp_max_array_or_list_width=80
```

Comment:
@nojaf It does feel to me that these defaults can be wider for F# code, likewise the other settings here

Reply:
I've opened https://github.com/fsprojects/fantomas/issues/2288 to discuss this.